xcell/cls/theory.py

```diff
@@ -119,10 +119,49 @@ def compute_tracer_ccl(self, name, tracer, mapper):
                 'with_hm': with_hm,
                 'normed': normed_profile}
 
+    def get_ccl_tkka(self, ccl_trA1, ccl_trA2, ccl_trB1, ccl_trB2,
```

Comment:
Can you add a comment saying what is tkka?

Reply:
Done